content/docs/command-reference/studio/login.md

```diff
@@ -0,0 +1,31 @@
+## studio login
+
+Authorize DVC with Studio to save the token to global [DVC configuration].
+
+[dvc configuration]:
+  /doc/user-guide/project-structure/configuration#config-file-locations
+
+## Synopsis
+
+```usage
+usage: dvc studio login [-h] [-q | -v] [-H HOSTNAME] [-s SCOPES] [-n NAME] [-d]
+```
+
+## Description
+
+By default, this command authorize dvc with Studio with default scopes and a
+random name as token name.
+
+## Options
+
+- `-h`, `--help` - show this help message and exit
+- `-q`, `--quiet` - Be quiet.
+- `-v`, `--verbose` - Be verbose.
+- `-H HOSTNAME`, `--hostname HOSTNAME` - The hostname of the Studio instance to
+  authenticate with.
+- `-s SCOPES`, `--scopes SCOPES` - The scopes for the authentication token.
```

Comment:
we need to describe them (or hide for now as I was suggesting).

Reply:
Lets decide the description on 
https://github.com/iterative/studio/issues/8161#issuecomment-1805824426 .
We can add that as followup or in this same PR after that.